Library/Formula/center-im.rb

```diff
@@ -21,7 +21,14 @@ def install
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
                           "--disable-msn"
-    system "make install"
+    system "make", "install"
+
+    # /bin/gawk does not exist on OS X
+    inreplace bin/"cimformathistory", "/bin/gawk", "/usr/bin/awk"
```

Comment:
Their bug reporter seems to be dead, but I'll try and find an email address somewhere or something.